src/migratePivotFolder.ts

```diff
@@ -0,0 +1,179 @@
+import { ContentRecord } from "@activeviam/activeui-sdk";
+import xml2js from "xml2js";
+import _cloneDeep from "lodash/cloneDeep";
+import { generateId } from "./generateId";
+
+/**
+ * Extracts and parses the XML calculated measure objects from the given pivot folder.
+ */
+const getCalculatedMeasuresFromPivotFolder = async (
+  pivotFolder: ContentRecord
+): Promise<
```

Comment:
I think this is confusing: the calculated measures live under /pivot/entitlements/cm, not /pivot.
So either I would keep the current function name, but then implement it in a way where `pivotFolder` is really /pivot, or keep the current implementation, but adapt the function name and argument:
```suggestion
const getCalculatedMeasures = async (
  calculatedMeasuresFolder: ContentRecord
): Promise<
```

You can choose the option you prefer.